ui/app/src/components/FormsEngine/controls/ImagePicker.tsx

```
@@ -26,85 +25,298 @@ import { DeleteOutlined, DownloadOutlined, EditOutlined } from '@mui/icons-mater
-						sx={{ width: '40%' }}
-						image={`${guestBase}${value}`}
-						alt="Live from space album cover"
+		<>
+			<Menu
+				anchorEl={addMenuButtonRef.current}
+				open={addMenuOpen}
+				onClose={() => setAddMenuOpen(false)}
+				sx={{
+					[`.${menuItemClasses.root}`]: { pl: 3 }
+				}}
+				children={menuOptions}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)